### PR TITLE
Home page is now respective to each role

### PR DESCRIPTION
**Pull request by Taz-Z**

### CODE REVIEW

**Review by br0wncrust4cean — APPROVED** [review]

File: Project/src/main/resources/templates/default.html [hunk 1]
```diff
@@ -2,19 +2,10 @@
...
-
-    <h2>User Info</h2>
-    <table>
-        <tr th:each="user : ${user}">
-            <td th:text="${user.username}"></td>
-            <td th:text="${user.password}"></td>
-            <td th:text="${user.role.toString()}"></td>
-        </tr>
-    </table>
+    <h1>Hello! Your account seems to be bugged - Please register again!</h1>
```
Comment (DanGravel) [inline Project/src/main/resources/templates/default.html]: if this page is not necessary just throw a 500 error

File: Project/src/main/java/app/RegisterController.java [hunk 2]
```diff
@@ -47,10 +48,18 @@ public String newUser(@ModelAttribute User user){
...
-        return "default";
+        switch(user.getRole()) {
+            case EDITOR:
+                return "redirect:/editor";
+            case REVIEWER:
+                return "redirect:/reviewer";
+            case SUBMITTER:
+                return "redirect:/upload";
+            default:
+                return "default";
```
Comment (DanGravel) [inline Project/src/main/java/app/RegisterController.java]: I think this should throw an error code

File: Project/src/main/resources/templates/default.html [hunk 3]
```diff
@@ -2,23 +2,11 @@
...
-            <td th:text="${user.username}"></td>
-            <td th:text="${user.password}"></td>
-            <td th:text="${user.role.toString()}"></td>
-        </tr>
-    </table>
-
-    <form th:action="@{/logout}" method="post">
-        <input type="submit" value="Sign Out"/>
-    </form>
+    <h1>Hello! Your account seems to be bugged - Please register again!</h1>
```
Comment (liamhickey) [inline Project/src/main/resources/templates/default.html]: Maybe we should consider throwing an error/going to an error page instead? This view isn't really all that useful anymore.

File: Project/src/main/resources/templates/default.html (hunk 1 above)
Comment (Taz-Z) [reply Project/src/main/resources/templates/default.html]: Sounds good

File: Project/src/main/java/app/RegisterController.java (hunk 2 above)
Comment (Taz-Z) [reply Project/src/main/java/app/RegisterController.java]: Will make change

File: Project/src/main/resources/templates/default.html (hunk 3 above)
Comment (Taz-Z) [reply Project/src/main/resources/templates/default.html]: Youre right

**Review by DanGravel — APPROVED** [review]

**Pull request merged**